src/Compiler/ES/Node.idr

```diff
@@ -54,8 +54,8 @@ compileExpr :
 compileExpr c s tmpDir outputDir tm outfile =
   do es <- compileToNode c s tm
      let out = outputDir </> outfile
-     Core.writeFile out es
-     coreLift_ $ chmodRaw out 0o755
+     writeFile out es
+     handleFileError out $ chmodRaw out 0o755
      pure (Just out)
```

Comment:
This can never return `Nothing`, can it? So we may as well change the type (and it saves
us the boggy error handling in `executeExpr`).

Reply:
Ugh, already fixed in a further commit